### PR TITLE
Release 1.4 ✨

### PR DESCRIPTION
**Pull request by lwasser**
# Stravalib Release Pull Request Template

* [An overview of our release workflow can be found in our documentation.](https://stravalib.readthedocs.io/contributing/build-release-guide.md)
* [Before making a release be sure to check out test PyPI](https://pypi.org/project/stravalib/) to ensure that the build is working properly.

## Release checklist
- [x] Be sure to clearly specify what version you are bumping to in the PR title: Example: Bump to version x.x
- [x] Add the version of this release to our changelog
- [x] Organize items changes under the new version in groups as follows: Added, Fixed and Changed
- [x] Add all contributors to the release below those sections
- [x] Wait for a maintainer to approve the pull request. Then merge! You are now ready to create the release.

Once this PR is merged you are ready to

- [ ] Create a tagged release on GitHub using the same version that you merged in the changelog added here
- [ ] When you publish that release, the GitHub action to push to PyPI will be invoked.

### CODE REVIEW

**Review by yihong0618 — APPROVED** [review]
LGTM

**Review by enadeau — APPROVED** [review]
Looks good to me

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -37,13 +36,13 @@ tests to support whatever fix of feature you're implementing.
...
-ping one of the maintainers here!*
+_If you are having a hard time getting the tests to run correctly feel free to
+ping one of the maintainers here!_
 
 <!---
 If you are a stravalib maintainer submitting a PR in preparation for a new release
 you can use the pull request release template:
 
-[https://github.com/stravalib/stravalib/compare/master...branch-name-here?template=release-pull-request-template.md](https://github.com/stravalib/stravalib/compare/master...branch-name-here?template=release-pull-request-template.md). Be sure to modify
+[https://github.com/stravalib/stravalib/compare/main...branch-name-here?template=release-pull-request-template.md](https://github.com/stravalib/stravalib/compare/master...branch-name-here?template=release-pull-request-template.md). Be sure to modify
```
Comment (lwasser) [inline .github/PULL_REQUEST_TEMPLATE.md:46]:
```suggestion
[https://github.com/stravalib/stravalib/compare/main...branch-name-here?template=release-pull-request-template.md](https://github.com/stravalib/stravalib/compare/main...branch-name-here?template=release-pull-request-template.md). Be sure to modify
```

**Comment by lwasser** [comment]
merging this and will create a release.

**Pull request merged**